app/controllers/catalog_controller.rb

```diff
@@ -71,14 +71,14 @@ class CatalogController < ApplicationController
     # ------------------------------------------------------------------------------------------
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
-    config.add_index_field solr_name("repository",        :stored_sortable), label: "Library", helper_method: :render_repository_facet_link
-    config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :highlight => true, :helper_method => :render_field_item
-    config.add_index_field solr_name("abstract",          :displayable),  :label => "Abstract", :highlight => true, :helper_method => :render_field_item
+
     config.add_index_field solr_name("format",            :displayable),  :label => "Format", :helper_method => :render_field_item
-    config.add_index_field solr_name("language",          :displayable),  :label => "Language", :helper_method => :render_field_item
-    config.add_index_field solr_name("unitdate",          :displayable),  :label => "Dates", :helper_method => :render_field_item
+    config.add_index_field solr_name("heading",           :displayable),  :label => "Contained in", :highlight => true, :helper_method => :render_series_facet_link
```

Comment:
@eshadatta Since you don't want this to be displayed for collections but it will always display when there is a heading value, which  there always is, you will want to add a conditional here, like so:

```ruby
config.add_index_field solr_name("heading", :displayable),  label: "Contained in", helper_method: :render_series_facet_link, unless: :is_collection?
```

Where you've added the following helper:

```ruby
  def is_collection?(field_config, doc)
    doc.is_archival_collection?
  end
```